src/Documentation/glossary.js

```diff
@@ -42,8 +42,17 @@ export default {
       name: 'Output',
       match: ['output', 'outputs'],
       desc:
-        'A file or a directory that is under DVC control. See `dvc add`,' +
-        ' `dvc run`, `dvc import`, `dvc import-url` commands.'
+        'A file or a directory that is under DVC control. See `dvc add`, ' +
+        '`dvc run`, `dvc import`, `dvc import-url` commands.'
+    },
+    {
+      name: 'External Dependency',
+      match: ['external dependency', 'external dependencies'],
+      desc:
+        'A DVC-file dependency with origin in an external source, for ' +
```

Comment:
should it include Git repo as a source?

Reply:
Hmmm how about "or even other DVC repositories" ? Updated in dc439ac

Reply:
sounds good to me!